### PR TITLE
[Keyboard] Add jolofsor folder and denial75 subfolder under QMK keyboards

### PR DESCRIPTION
**Pull request by jolofsor**
I have made my personal keyboard and would like to have it added in the QMK master repository. 

## Description

This will create a folder of jolofsor with a subfolder of denial75 that contains all the keyboard files.  I have tested these files already and I have also gone through the PR checklist to see if the code complies based on my understanding of the checklist. 

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Comment by jolofsor** [comment]
Closed and Reopened as I noticed a typo in info.json

File: keyboards/jolofsor/denial75/config.h [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#ifndef CONFIG_H
+#define CONFIG_H
```
Comment (drashna) [inline keyboards/jolofsor/denial75/config.h:18]:
```suggestion
```

File: keyboards/jolofsor/denial75/info.json [hunk 2]
```diff
@@ -0,0 +1,101 @@
+{
+    "keyboard_name": "denial75",
+    "url": "",
+    "maintainer": "jolofsor",
+    "width": 16.5,
+    "height": 6.5,
```
Comment (drashna) [inline keyboards/jolofsor/denial75/info.json:6]:
```suggestion
```

File: keyboards/jolofsor/denial75/rules.mk [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+MCU = atmega32u4
+
+# Bootloader selection
+#   Teensy       halfkay
+#   Pro Micro    caterina
+#   Atmel DFU    atmel-dfu
+#   LUFA DFU     lufa-dfu
+#   QMK DFU      qmk-dfu
+#   ATmega32A    bootloadHID
+#   ATmega328P   USBasp
```
Comment (drashna) [inline keyboards/jolofsor/denial75/rules.mk:11]:
```suggestion
```

File: keyboards/jolofsor/denial75/config.h [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+/* RGB Definitions */
+
+#define RGB_DI_PIN D2
+#define RGBLED_NUM 84
+#define RGBLIGHT_MAX_LAYERS 32
+#define RGBLIGHT_HUE_STEP 10
+#define RGBLIGHT_SAT_STEP 17
+#define RGBLIGHT_VAL_STEP 17
+#define RGBLIGHT_LIMIT_VAL 200
+#endif
```
Comment (drashna) [inline keyboards/jolofsor/denial75/config.h:66]:
```suggestion
```

**Review by drashna — CHANGES_REQUESTED** [review]

**Comment by jolofsor** [comment]
I have commited the changes requested by Drashna. I have also reverified that the code can still be compiled and flashed to keyboard succesfully

**Comment by jolofsor** [comment]
Have committed my via files in this PR as well

File: keyboards/jolofsor/denial75/config.h [hunk 5]
```diff
@@ -0,0 +1,63 @@
...
+/* Mechanical locking support. Use KC_LCAP, KC_LNUM or KC_LSCR instead in keymap */
+#define LOCKING_SUPPORT_ENABLE
+
+/* Locking resynchronize hack */
+#define LOCKING_RESYNC_ENABLE
+
+
+/* RGB Definitions */
+
+#define RGB_DI_PIN D2
```
Comment (drashna) [inline keyboards/jolofsor/denial75/config.h:57]: No animations enabled by default?

Comment (jolofsor) [reply keyboards/jolofsor/denial75/config.h:57]: Yes. I would be keeping this one in static light only.

**Review by drashna — APPROVED** [review]

File: keyboards/jolofsor/denial75/rules.mk [hunk 6]
```diff
@@ -0,0 +1,19 @@
...
+BOOTMAGIC_ENABLE = no	# Virtual DIP switch configuration
+MOUSEKEY_ENABLE = no	# Mouse keys
+EXTRAKEY_ENABLE = yes	# Audio control and System control
+CONSOLE_ENABLE = no	# Console for debug
+COMMAND_ENABLE = no    # Commands for debug and configuration
+SLEEP_LED_ENABLE = no  # Breathing sleep LED during USB suspend
+NKRO_ENABLE = yes	# USB Nkey Rollover - if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
+BACKLIGHT_ENABLE = no  # Enable keyboard backlight functionality
+AUDIO_ENABLE = no
+RGBLIGHT_ENABLE = yes
```
Comment (fauxpark) [inline keyboards/jolofsor/denial75/rules.mk:19]:
```suggestion
#   change yes to no to disable
#
BOOTMAGIC_ENABLE = no       # Enable Bootmagic Lite
MOUSEKEY_ENABLE = no        # Mouse keys
EXTRAKEY_ENABLE = yes       # Audio control and System control
CONSOLE_ENABLE = no         # Console for debug
COMMAND_ENABLE = no         # Commands for debug and configuration
# Do not enable SLEEP_LED_ENABLE. it uses the same timer as BACKLIGHT_ENABLE
SLEEP_LED_ENABLE = no       # Breathing sleep LED during USB suspend
# if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
NKRO_ENABLE = yes           # USB Nkey Rollover
BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality
RGBLIGHT_ENABLE = yes       # Enable keyboard RGB underglow
AUDIO_ENABLE = no           # Audio output
```

File: keyboards/jolofsor/denial75/config.h [hunk 7]
```diff
@@ -0,0 +1,63 @@
...
+
+#pragma once
+#include "config_common.h"
+
+/* USB Device descriptor parameter */
+#define VENDOR_ID       0x4A48
+#define PRODUCT_ID      0x0001
+#define DEVICE_VER      0x0001
+#define MANUFACTURER    jsor-hpoli
+#define PRODUCT         "denial75"
```
Comment (fauxpark) [inline keyboards/jolofsor/denial75/config.h:25]:
```suggestion
#define PRODUCT         denial75
```

File: keyboards/jolofsor/denial75/keymaps/via/keymap.c [hunk 8]
```diff
@@ -0,0 +1,134 @@
...
+	),
+
+	[1] = LAYOUT_denial75_ansi(
+	    KC_NO,   KC_NO,   KC_NO,   KC_NO,  KC_NO,  KC_NO,  KC_NO,  KC_NO,  KC_NO,  KC_NO,   KC_NO,   KC_NO,   KC_NO,   KC_MUTE, KC_AUDIO_VOL_DOWN, KC_AUDIO_VOL_UP,
+    	KC_NO,   KC_NO,   KC_NO,   KC_NO,  KC_NO,  KC_NO,  KC_NO,  KC_NO,  KC_NO,  KC_NO,   KC_NO,   KC_NO,   KC_NO,             KC_NO,    KC_BRIU,
+    	KC_NO,            LEDRED,  LEDBLUE,  LEDYELLOW,  KC_NO,  KC_NO,  KC_NO,  KC_NO,  KC_NO,   KC_NO,   KC_NO,   KC_NO,   KC_NO,    KC_NO,    KC_BRID,
+    	KC_NO,  	      LEDORANGE,   LEDGREEN,  LEDPURPLE,  KC_NO,  KC_NO,  KC_NO,  KC_NO,  KC_NO,   KC_NO,   KC_NO,   KC_NO,   KC_NO,              RGB_VAI,
+   		 KC_NO,  LEDWHITE,   LEDPINK,  LEDBLACK,  KC_NO,  KC_NO,  KC_NO,  KC_NO,  KC_NO,   KC_NO,   KC_NO,            KC_NO,    KC_NO,    RGB_VAD,
+    	KC_NO,   KC_NO, 	   KC_NO,                  KC_NO,                           KC_NO,   MO(1),   RGB_TOG, KC_NO,    KC_NO,    KC_NO
+	)
```
Comment (fauxpark) [inline keyboards/jolofsor/denial75/keymaps/via/keymap.c:133]: Need two more empty layers here.

**Review by fauxpark — CHANGES_REQUESTED** [review]

File: keyboards/jolofsor/denial75/keymaps/via/keymap.c (hunk 8 above)
Comment (jolofsor) [reply keyboards/jolofsor/denial75/keymaps/via/keymap.c:133]: @fauxpark do you mean add 2 more layers with KC_NO for all the keys?

Comment (fauxpark) [reply keyboards/jolofsor/denial75/keymaps/via/keymap.c:133]: Yes. `DYNAMIC_KEYMAP_LAYER_COUNT` is 4 by default, so you need to either define 4 layers in the via keymap, or redefine it to however many layers you want (or will fit in the MCU's EEPROM).

Comment (jolofsor) [reply keyboards/jolofsor/denial75/keymaps/via/keymap.c:151]: @fauxpark Thanks for the explanation. I have now added in two empty layers to the via keymap.c based on your input. requesting for review.

**Comment by jolofsor** [comment]
@fauxpark following up on your review of my changes based on your comments

File: keyboards/jolofsor/denial75/rules.mk [hunk 9]
```diff
@@ -0,0 +1,21 @@
...
+# Build Options
+#   change yes to no to disable
+#
+BOOTMAGIC_ENABLE = no       # Enable Bootmagic Lite
+MOUSEKEY_ENABLE = no        # Mouse keys
+EXTRAKEY_ENABLE = yes       # Audio control and System control
+CONSOLE_ENABLE = no         # Console for debug
+COMMAND_ENABLE = no         # Commands for debug and configuration
+# Do not enable SLEEP_LED_ENABLE. it uses the same timer as BACKLIGHT_ENABLE
+SLEEP_LED_ENABLE = no       # Breathing sleep LED during USB suspend
```
Comment (fauxpark) [inline keyboards/jolofsor/denial75/rules.mk:16]:
```suggestion
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

**Review by fauxpark — APPROVED** [review]

**Pull request merged**